extensions/wikia/AdEngine/js/spec/lookup/services.spec.js

```diff
@@ -2,18 +2,19 @@
 describe('Method ext.wikia.adEngine.lookup.services', function () {
 	'use strict';
 
+	// TODO tests
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[project-id]: wikia:app